bazel/envoy_build_system.bzl

```diff
@@ -369,3 +374,10 @@ def envoy_select_google_grpc(xs, repository = ""):
         repository + "//bazel:disable_google_grpc": [],
         "//conditions:default": xs,
     })
+
+# Select stuff
```

Comment:
This comment isn't very descriptive :) Probably best to say what exported symbols are..